.templates/new-module/src/js/react.jsx

```diff
@@ -0,0 +1,18 @@
+/*! [replace-name] v[replace-version] */
```

Comment:
wondering if you've considered the case where we might need more than a singular file per framework paradigm?